{"name": "Seleccionar local", "status": "failed", "statusDetails": {"message": "AssertionError: False is not true : Ocurrió un error al confirmar la tienda, posiblemente no esté disponible", "trace": "self = <Login.LoginAccount testMethod=test_05_login_select>\n\n    @allure.title(u\"Seleccionar local\")\n    @allure.description(u\"Se requiere seleccionar un local, en este caso en CENTRO CIVICO\")\n    @allure.story(u'Dirección')\n    def test_05_login_select(self):\n        self.test_02_login()\n        #Opcion 3: Recojo en tienda, Seleccionar tienda\n        with allure.step(u\"Seleccionamos recojo en tienda\"):\n            elem = self.driver.find_elements(By.CSS_SELECTOR,'div.button-delivery')[1]\n            elem.click()\n            #Seleccionar el item de tienda de Cercado de Lima\n            elem = self.driver.find_element(By.XPATH,'//h4[contains(text(),\"Cercado de Lima\")]')\n        with allure.step(u\"Confirmamos la selección\"):\n            self.driver.execute_script(\"arguments[0].click();\", elem)\n            #elem = self.driver.find_element(By.CSS_SELECTOR,'div.buttons-store.btn')\n            elem = self.driver.find_element(By.XPATH,'//button[contains(text(),\"CONFIRMAR\")]')\n            elem.click()\n        with allure.step(u\"Validamos la tienda seleccionada\"):\n            #Verificar que se seleccionó la tienda\n            try:\n                element_present = EC.presence_of_element_located((By.CSS_SELECTOR, \"div.store-name > p\"))\n>               WebDriverWait(self.driver, 5).until(element_present)\n\nLogin.py:248: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"c93b0d4b284dff6166650b4ca1cc12e4\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x000001F2151DC280>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except InvalidSelectorException as e:\n                raise e\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tOrdinal0 [0x010169A3+2582947]\nE       \tOrdinal0 [0x00FAA6D1+2139857]\nE       \tOrdinal0 [0x00EA3A98+1063576]\nE       \tOrdinal0 [0x00ECFF3E+1244990]\nE       \tOrdinal0 [0x00ED013B+1245499]\nE       \tOrdinal0 [0x00EF9F52+1417042]\nE       \tOrdinal0 [0x00EE8594+1344916]\nE       \tOrdinal0 [0x00EF834A+1409866]\nE       \tOrdinal0 [0x00EE8366+1344358]\nE       \tOrdinal0 [0x00EC5176+1200502]\nE       \tOrdinal0 [0x00EC6066+1204326]\nE       \tGetHandleVerifier [0x011BBE02+1675858]\nE       \tGetHandleVerifier [0x0127036C+2414524]\nE       \tGetHandleVerifier [0x010ABB01+560977]\nE       \tGetHandleVerifier [0x010AA8D3+556323]\nE       \tOrdinal0 [0x00FB020E+2163214]\nE       \tOrdinal0 [0x00FB5078+2183288]\nE       \tOrdinal0 [0x00FB51C0+2183616]\nE       \tOrdinal0 [0x00FBEE1C+2223644]\nE       \tBaseThreadInitThunk [0x7530FA29+25]\nE       \tRtlGetAppContainerNamedObjectPath [0x76F97A9E+286]\nE       \tRtlGetAppContainerNamedObjectPath [0x76F97A6E+238]\n\nC:\\Users\\artur\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\selenium\\webdriver\\support\\wait.py:89: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <Login.LoginAccount testMethod=test_05_login_select>\n\n    @allure.title(u\"Seleccionar local\")\n    @allure.description(u\"Se requiere seleccionar un local, en este caso en CENTRO CIVICO\")\n    @allure.story(u'Dirección')\n    def test_05_login_select(self):\n        self.test_02_login()\n        #Opcion 3: Recojo en tienda, Seleccionar tienda\n        with allure.step(u\"Seleccionamos recojo en tienda\"):\n            elem = self.driver.find_elements(By.CSS_SELECTOR,'div.button-delivery')[1]\n            elem.click()\n            #Seleccionar el item de tienda de Cercado de Lima\n            elem = self.driver.find_element(By.XPATH,'//h4[contains(text(),\"Cercado de Lima\")]')\n        with allure.step(u\"Confirmamos la selección\"):\n            self.driver.execute_script(\"arguments[0].click();\", elem)\n            #elem = self.driver.find_element(By.CSS_SELECTOR,'div.buttons-store.btn')\n            elem = self.driver.find_element(By.XPATH,'//button[contains(text(),\"CONFIRMAR\")]')\n            elem.click()\n        with allure.step(u\"Validamos la tienda seleccionada\"):\n            #Verificar que se seleccionó la tienda\n            try:\n                element_present = EC.presence_of_element_located((By.CSS_SELECTOR, \"div.store-name > p\"))\n                WebDriverWait(self.driver, 5).until(element_present)\n                elem = self.driver.find_element(By.CSS_SELECTOR,'div.store-name > p')\n                self.screenshot(\"Seleccionar tienda\")\n                self.assertEqual(elem.text,\"CENTRO CIVICO\",\"No se seleccionó la tienda esperada\")\n            except TimeoutException:\n                self.screenshot(\"error_seleccion_tienda\")\n>               self.assertTrue(self.is_element_present(By.CSS_SELECTOR,'div.store-name > p'),'Ocurrió un error al confirmar la tienda, posiblemente no esté disponible')\nE               AssertionError: False is not true : Ocurrió un error al confirmar la tienda, posiblemente no esté disponible\n\nLogin.py:254: AssertionError"}, "description": "Se requiere seleccionar un local, en este caso en CENTRO CIVICO", "steps": [{"name": "Iniciar el controlador de Chrome", "status": "passed", "start": 1646234537494, "stop": 1646234538569}, {"name": "Accedemos a la página de inicio de sesión", "status": "passed", "start": 1646234538569, "stop": 1646234543246}, {"name": "Ingresamos las credenciales de inicio de sesión con cuenta nativa", "status": "passed", "start": 1646234543246, "stop": 1646234543361}, {"name": "Validamos el inicio de sesión", "status": "passed", "start": 1646234543361, "stop": 1646234548951}, {"name": "Seleccionamos recojo en tienda", "status": "passed", "start": 1646234548951, "stop": 1646234549845}, {"name": "Confirmamos la selección", "status": "passed", "start": 1646234549845, "stop": 1646234549888}, {"name": "Validamos la tienda seleccionada", "status": "failed", "statusDetails": {"message": "AssertionError: False is not true : Ocurrió un error al confirmar la tienda, posiblemente no esté disponible\n", "trace": "  File \"D:\\PROYECTOS CICLO IX\\SELENIUM\\PopeyesPruebas\\UnitTestPopeyes\\Login.py\", line 254, in test_05_login_select\n    self.assertTrue(self.is_element_present(By.CSS_SELECTOR,'div.store-name > p'),'Ocurrió un error al confirmar la tienda, posiblemente no esté disponible')\n  File \"C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.9_3.9.2800.0_x64__qbz5n2kfra8p0\\lib\\unittest\\case.py\", line 688, in assertTrue\n    raise self.failureException(msg)\n"}, "attachments": [{"name": "error_seleccion_tienda", "source": "6d781c4f-b843-4d96-a69a-e4da6d7ef010-attachment.png", "type": "image/png"}], "start": 1646234549888, "stop": 1646234580542}], "attachments": [{"name": "stdout", "source": "ab4ec2e3-f28a-48b2-9a41-6cca810c2768-attachment.txt", "type": "text/plain"}], "start": 1646234537494, "stop": 1646234582646, "uuid": "bcc62d60-c7f5-47f2-850a-a1dab8fdcef4", "historyId": "c13e10eb82851ddb98dea5d8d7323156", "testCaseId": "014b73548cfad7c1382a6f0c60edf3c5", "fullName": "Login.LoginAccount#test_05_login_select", "labels": [{"name": "story", "value": "Dirección"}, {"name": "feature", "value": "Log in"}, {"name": "suite", "value": "Login"}, {"name": "subSuite", "value": "LoginAccount"}, {"name": "host", "value": "DESKTOP-FT84D78"}, {"name": "thread", "value": "8548-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Login"}]}